distribution/src/config/jvm.options

```diff
@@ -16,19 +33,22 @@
 ##
 ################################################################
 
-# Xms represents the initial size of total heap space
-# Xmx represents the maximum size of total heap space
+# Xms represents the initial size of the JVM heap
+# Xmx represents the maximum size of the JVM heap
 
 -Xms${heap.min}
 -Xmx${heap.max}
```

Comment:
I wonder if we should take this further and remove this from this file. The way I envision this working is that in `JvmOptionsParser`, we'd emit a `-Xms1g` and/or `-Xmx1g` if `Xms` (`MinHeapSize`) or `Xmx` (`MaxHeapSize`) are not set respectively.

Reply:
Yes; does the same reasoning apply to all the other settings in this file too?

However I don't have the capacity to take this work on myself at the moment.

Reply:
I can pick this up - I'll raise a new issue to move the default heap settings.

Reply:
I raised #61514.